src/app/components/Recommendations/RecommendationsPromoImage/index.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render } from '../../../../components/react-testing-library-with-providers';
+import { render } from '#app/components/react-testing-library-with-providers';
```

Comment:
is this switch to an aliased path to follow our pattern as a result of the `.tsx` conversion?

Reply:
I'd personally use it with normal JS as well, but yes we're going with aliasing moving forward.

Reply:
Cool, I remember a while back we had issues with webpack(?) or cypress(?) recognising the aliased routes. I think it was mainly around the time we migrated psammead in so was jw. Good to know! I find the aliased routes more readable tbh